src/components/shared/ModalRegistration/AuthForm/index.tsx

```diff
@@ -28,15 +29,16 @@ const AuthForm: FC<AuthProps> = ({ isModalOpen, setIsModalOpen }) => {
 
     dispatch(thunkAuthSignin({ email: login, password })).then(res => {
       dispatch(setIsLogged(true));
+      //Для Покупця
       if (res.payload.buyer) {
-        dispatch(setLastName(res.payload.buyer.lastName));
-        dispatch(setName(res.payload.buyer.name));
+        dispatch(SET_LOGGED_USER(res.payload.buyer));
+      }
+      // Для Продавця
+      if (res.payload.seller) {
+        dispatch(SET_LOGGED_USER(res.payload.seller));
       }
-      /*дописать для продавца*/
```

Comment:
Write comments in English, please. We can see payload.buyer and payload.seller. Why we need comments for this? I think is better to remove first and second comments. About the last one comment - use keywords to show that you need to do something with that in future //TODO or //todo would be great and if you have IDEA (Webstorm etc.) you can see info about todo when you create commit with this file next time.

Reply:
agree my fault, I saw not in English and without any mind written in Ukrainian also